### ISSUE TITLE
Getting null safety issue

### ISSUE THREAD

**Issue by kamleshwebtech** [rafalbednarczuk/curved_navigation_bar#101]
I am setup 0.3.4 version of curved_navigation_bar but getting NULL SAFETY error as:

```
The library 'package:curved_navigation_bar/curved_navigation_bar.dart' is legacy, and should not be imported into a null safe library.
Try migrating the imported library
```

Kindly suggest what should I do to fix this issue. Thanks. a lot.

**Comment by kevin4dhd** [comment]
the same error

Error: Cannot run with sound null safety, because the following dependencies
don't support null safety:

 - package:curved_navigation_bar

**Comment by rafalbednarczuk** [comment]
fixed in 1.0.0
